src/components/core/pivotTable/AggregationsMenu.tsx

```diff
@@ -18,6 +21,9 @@ import { AVAILABLE_TOTALS } from "../../visualizations/table/totals/utils";
 import AggregationsSubMenu from "./AggregationsSubMenu";
 import menuHelper from "./aggregationsMenuHelper";
 import { FIELD_TYPE_ATTRIBUTE } from "./agGridConst";
+import { SHOW_DELAY_DEFAULT, HIDE_DELAY_DEFAULT } from "../../../internal/constants/bubble";
+
+const NATIVE_TOTAL = "nat";
```

Comment:
Don't we have this constant somewhere else already?

Reply:
I searched but did not found it.